xrspatial/aspect.py

```diff
@@ -154,21 +154,52 @@ def _run_dask_numpy(data:da.Array) -> da.Array:
 
 
 def aspect(agg: xr.DataArray, name:str ='aspect'):
-    """Returns downward slope direction in compass degrees (0 - 360) with 0 at 12 o'clock.
+    """Calculates, for all cells in the array, the downward slope direction of each cell 
+        based on the elevation of its neighbors in a 3x3 grid. The value is measured 
+        clockwise in degrees with 0 and 360 at due north. Flat areas are given a value of -1.
 
     Parameters
     ----------
     agg : DataArray
+        Numpy, CuPy, Dask, or CuPy GPU data array with agg.data = elevation
```

Comment:
The explanation for param `agg` and the return of the function should be more clear. Ex: agg: xarray.DataArray. The data dimension of input aggregate is 2D and can be either a numpy array, a cupy array, a dask array with numpy backed or a dask array with cupy backed...